deploy/operator.yaml

```
@@ -41,6 +41,13 @@ spec:
       containers:
         - name: noobaa-operator
           image: NOOBAA_OPERATOR_IMAGE
+          args:
+            - operator  
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)